src/utils/slugModifiers/isSlugLegacy.ts

```diff
@@ -0,0 +1,7 @@
+export const LEGACY_PROGRAMME_SUFFIX = "-l";
```

Comment:
Perhaps this could just be `LEGACY_SLUG_SUFFIX` because it's used for both programme and subject